src/model/table/table-types.ts

```diff
@@ -0,0 +1,12 @@
+import { TableDocument } from './table-definitions';
+import { Article } from '../documents/article';
+import { Recipe } from '../documents/recipe';
```

Comment:
Please change imports to absolute path (just from /src directory, so probably remove only ../../).

Reply:
Fixed.